src/lib/resource.test.js

```diff
@@ -18,10 +18,10 @@ const urlReturn = `
 <meta name="national-centers" content="Health, Behavioral Health, and Safety" />
 <meta name="node-id" content="7858" />
 <meta name="exclude-from-dynamic-view" content="False" />
-<script>window.dataLayer = window.dataLayer || []; window.dataLayer.push({"language":"en","country":"US","siteName":"ECLKC","entityLangcode":"en","entityVid":"326638","entityCreated":"1490966152","entityStatus":"1","entityName":"leraa","entityType":"node","entityBundle":"page_front","entityId":"2212","entityTitle":"Head Start","userUid":0});</script>
-<link rel="canonical" href="https://eclkc.ohs.acf.hhs.gov/" />
-<link rel="image_src" href="https://eclkc.ohs.acf.hhs.gov/themes/gesso/images/site-logo.png" />
-<title>Head Start | ECLKC</title>
+<script>window.dataLayer = window.dataLayer || []; window.dataLayer.push({"language":"en","country":"US","siteName":"HeadStart","entityLangcode":"en","entityVid":"326638","entityCreated":"1490966152","entityStatus":"1","entityName":"leraa","entityType":"node","entityBundle":"page_front","entityId":"2212","entityTitle":"Head Start","userUid":0});</script>
```

Comment:
it is weird to include their google analytics code in our test and then feel we need to keep it updated

Reply:
Done.